package.json

```diff
@@ -5,7 +5,9 @@
     "main": "index.js",
     "scripts": {
         "start": "webpack-dev-server --mode development --open --hot",
-        "build": "webpack --mode production"
+        "build": "webpack --mode production",
+        "test": "jest --watchAll",
```

Comment:
usually `test` script is used for single run

to have watch you can add one more smth like `test:watch`

Reply:
Updated